meetings/2021-11-22.md

```diff
@@ -0,0 +1,93 @@
+* AC: I've look into the `azp` claim in the past. It is a good thing to have. I do not recall why we decided to use it. It seems like a good fit, so we should look more into it. It looks like it would desambiguate the claims we use.
+* eP: If we want to issue ID tokens scoped to a specific AuthZ server, then this information would be in the `aud` claim, and it would be good not to rely on this claim to discover the client ID.
+* eP: The PR does not include the discovery of the AuthZ server. Each actor presened in the diagram should be a conformance class so that they are individually described by the spec, in a slightly different structure. 
+* AC: +1 on reorganizing to frame the conformance classes.
+* eP: The restructuring could happen in a different PR, but in the current PR should include a description of the AuthZ server responsabilities, and examples of claims in the ID token. The fact that all of this is extensible by othe specs
+* AC: What approach were you thinking about for the authZ discovery ? including but not restricting to UMA as_uri ?
+* eP: Yes. I don't have other examples besides UMA, but there may be others.
+* AC: We want to be clear about expectations not to lock ourselves in a box. We should write something along the lines: the `as_uri` parameter must be included in the WWW-Authenticate header, unless the active authentication scheme requires a different discovery mechanism. 
+* eP: /Updates issue 25 accordingly/. Ideally we should merge this by the end of the month, but we can defer pain points in the existing issues.
+* AC: since this is a big change, we should make sure people are well-aligned on this. Should we reach out to CSS/NSS maintainers while this is still a draf, so that we have early feedback ?
+* eP: It would be nice to have some way of linking to the rendered preview of that PR for discussion
+* AC: We could do this manually if automating is too much of a hassle.
+* eP: We can move on with the discussion until we are happy enough with it and open it for broader discussion, and only then render it.
+* AC: It would be good to have a clear horizon for the public feedback. We should be conscious about the fact that the Christmas holidays are coming up and that a lot of people are going to be out at this point.
+* eP: Worst case scenario, if feedback is too negative with compelling arguments, we can always revert the change. We could agree on merging this this week, and open a 2-week window.
+* AC: This sounds like a good plan. This is also a good first step to define a step forward, because we stop specifying the role of the OIDC issuer issuing Access Token, but we don't prevent it altogether, which maintians backwards compat
```

Comment:
```suggestion
* AC: This sounds like a good plan. This is also a good first step to define a step forward, because we stop specifying the role of the OIDC issuer issuing Access Token, but we don't prevent it altogether, which maintains backwards compatibility
```